_events/2017-01-02-matthew-at-agileonthebench.md

```diff
@@ -0,0 +1,20 @@
+---
+author: 
+ - Matthew Butt 
+speaker: Matthew Butt
+layout: event
+name: matthew-at-agile-on-the-bench 
+location: "Arnold Circus, London E2 7JS, UK"
+image:
+    src: /assets/img/custom/events/agile-on-the-bench.png 
```

Comment:
Also, I think we need to crop whatever image we use, as it's been squashed horizontally to fit the page.